PEReaderAndWriter/PEReader/PEFileStructures.cs

```diff
@@ -152,8 +152,8 @@ internal enum COR20Flags : uint {
     Bit32Required = 0x00000002,
     ILLibrary = 0x00000004,
     StrongNameSigned = 0x00000008,
-    Prefers32bits = 0x00000010,
```

Comment:
It might be good to put in NativeEntrypoint here (the correct flag for this value). That would let you keep the commented out code in GetEntryPointMethod.

Reply:
Okay, can do. What is the process to update my PR?

Reply:
AFAIK you should be able to publish another commit on this branch with modifications pertaining to the changes discussed in the review. I admit that my own experience with complicated multi-stage reviews is very limited so we might need some expert opinion on this.